docs/docs/guides/automate/sensors/testing-run-status-sensors.md

```diff
@@ -30,7 +30,7 @@ Then we can execute this job and pull the attributes we need to build the `conte
 />
 
 {/* TODO the methods and statuses below do not exist in API docs */}
-{/* We have provided convenience functions <PyObject section="execution" module="dagster" object="ExecuteInProcessResult" method="get_job_success_event" /> and <PyObject section="execution" module="dagster" object="ExecuteInProcessResult" method="get_job_failure_event" /> for retrieving `DagsterRunStatus.SUCCESS` and `DagsterRunStatus.FAILURE` events, respectively. If you have a run status sensor triggered on another status, you can retrieve all events from `result` and filter based on your event type. */}
+{/* We have provided convenience functions <PyObject section="execution" module="dagster" object="ExecuteInProcessResult" method="get_run_success_event" /> and <PyObject section="execution" module="dagster" object="ExecuteInProcessResult" method="get_run_failure_event" /> for retrieving `DagsterRunStatus.SUCCESS` and `DagsterRunStatus.FAILURE` events, respectively. If you have a run status sensor triggered on another status, you can retrieve all events from `result` and filter based on your event type. */}
```

Comment:
```suggestion
We have provided convenience functions <PyObject section="execution" module="dagster" object="ExecuteInProcessResult.get_run_success_event" /> and <PyObject section="execution" module="dagster" object="ExecuteInProcessResult.get_run_failure_event" /> for retrieving `DagsterRunStatus.SUCCESS` and `DagsterRunStatus.FAILURE` events, respectively. If you have a run status sensor triggered on another status, you can retrieve all events from `result` and filter based on your event type.
```

Reply:
`method` isn't supported in the new `PyObject` component, so I think you'll just need to attach the method to the object, as suggested.